python/ray/autoscaler/node_provider.py

```diff
@@ -9,33 +9,28 @@
 
 
 def import_aws():
-    from ray.autoscaler.aws.config import bootstrap_aws
     from ray.autoscaler.aws.node_provider import AWSNodeProvider
-    return bootstrap_aws, AWSNodeProvider
+    return AWSNodeProvider.bootstrap_config, AWSNodeProvider
 
 
 def import_gcp():
-    from ray.autoscaler.gcp.config import bootstrap_gcp
     from ray.autoscaler.gcp.node_provider import GCPNodeProvider
-    return bootstrap_gcp, GCPNodeProvider
+    return GCPNodeProvider.bootstrap_config, GCPNodeProvider
 
 
 def import_azure():
-    from ray.autoscaler.azure.config import bootstrap_azure
     from ray.autoscaler.azure.node_provider import AzureNodeProvider
-    return bootstrap_azure, AzureNodeProvider
+    return AzureNodeProvider.bootstrap_config, AzureNodeProvider
 
 
 def import_local():
-    from ray.autoscaler.local.config import bootstrap_local
     from ray.autoscaler.local.node_provider import LocalNodeProvider
-    return bootstrap_local, LocalNodeProvider
+    return LocalNodeProvider.bootstrap_config, LocalNodeProvider
 
 
 def import_kubernetes():
-    from ray.autoscaler.kubernetes.config import bootstrap_kubernetes
     from ray.autoscaler.kubernetes.node_provider import KubernetesNodeProvider
-    return bootstrap_kubernetes, KubernetesNodeProvider
+    return KubernetesNodeProvider.bootstrap_config, KubernetesNodeProvider
```

Comment:
Should we remove the two-arg return, just 1 return?

Reply:
checkout if the last commit is what you had in mind.

Reply:
Actually, nevermind.
The problem is that external node provider is not known in advance. Let me fix that.

Reply:
Actually, the main problem is in the external node provider.